src/interfaces/aggregated-data.interface.ts

```diff
@@ -115,6 +116,63 @@ export const JAPANESE_ATTRIBUTE_NAME: Record<ObjectClassAttribute, string> = {
   prefectures: "都道府県",
   carCategories: "車両分類",
 } as const;
+export const REGIONS_PREFECTURES: Record<string, { name: string; prefectures: string[] }> = {
```

Comment:
```suggestion
export const REGIONS_PREFECTURES: Record<string, { name: string; prefectures: (keyof typeof PREFECTURES)[]  }> = {
```

こんなふうにするとprefecturesの配列の中に入る文字列を制限できてより良いと思います